proposals/graphql/proposal/src/main/asciidoc/chapters/about.asciidoc

```diff
@@ -37,27 +37,115 @@ Also see <<appendix_b>>
+* July 2015
+
+According to its own definition: "GraphQL is a query language for describing the capabilities and requirements of data models for client‐server applications." 
+
+It should be stressed that GraphQL is independant from any transport protocol and database technology:
+
+* it does not recommend the use of HTTP. However, in practice, and as with REST, it is clearly the most widely used protocol,
+* it isn't tied to any specific database or storage engine and is instead backed by existing code and data.
+
+[[what_make_graphql_different]]
+=== What make GraphQL different?
+In practice, here are the main differentiating features of GraphQL compared to REST:
+
+* *schema-driven*: a GraphQL API natively exposes a schema describing the structure of the data and operations (queries and mutations) exposed. This schema acts as a contract between the server and its clients. In a way GraphQL provides an explicit answer to the API discovery problem where REST relies on the ability of developers to use HATEOS,
+* *single HTTP endpoint*: a typical GraphQL API is made of a single endpoint and access to data and operations is achieved through the query language. In a HTTP context, the endpoint is defined as a URL and the query can be  transported as a query string (GET request) or in the request body (POST request),
+* *flexible data retrieval*: by construction the query language enables to select the expected data in the response with a fine level of granularity,
+* *reduction of server requests*: the language allows the client to aggregate the expected data into a single request,
+* *easier version management*: thanks to the native capabilities to create new data while deprecating old ones, 
+* *partial results*: partial results are delivered by the GraphQL server in case of errors. A GraphQL result is made of data and errors. It's not an all or nothing approach. How to process such a result is the responsability of the client,
+* *low coupling with HTTP*: GraphQL does not try to make the most of the HTTP semantic. Queries can be made using GET or POST requests. The HTTP result code  does not reflect the GraphQL response. This aspect is subject to discussion and perhaps GraphQL will have to evolve towards a better integration with HTTP to broaden its adoption,
+* *challenging authorization handling*: an appropriate data access authorization policy must be defined and implemented to counter the extreme flexibility of the query language. For example, a customer may request data that are authorized and others that are prohibited,
+* *challenging API management*: most API management solutions are based on REST capabilities and allow for URL-based policies to be established. As we have seen, a GraphQL API has a single entry point. An analysis of the request is necessary to establish a policy. For instance, to prevent the execution of an overly complex request that would crash the server.
+
+[[graphql_beyond-the-hype]]
+=== GraphQL beyond the hype
```

Comment:
The `GraphQL beyond the hype` section should probably go in the README rather than the spec document.  My opinion is that the spec document should not comment on controversy, popularity, etc. but rather should focus on the problems and how the spec/API solves them.  What do others think?

Reply:
ok